dataland-frontend/src/components/resources/frameworkDataSearch/DisplayFrameworkDataTable.vue

```diff
@@ -188,7 +194,7 @@ export default defineComponent({
      * @returns true if the string is 'Yes' or 'No'
      */
     isYesNo(value: string) {
-      return (Object.values(YesNo) as string[]).includes(value);
+      return Object.values(YesNo).toString().includes(value);
```

Comment:
Warum diese Änderung? Jetzt returnt isYesNo auch true wenn value auf `s,N` gesetzt wurde.

Reply:
This was changed due to a sonar issue. However i think you are right. I will revert it.